server/graphql/general/src/queries/program_settings.rs

```diff
@@ -25,39 +20,46 @@ pub struct ProgramRequisitionOrderTypeNode {
 
 #[derive(SimpleObject)]
 pub struct ProgramRequisitionSettingNode {
-    pub program: ProgramNode,
+    /// Program name
```

Comment:
the comments aren't terribly informative 🤷

Reply:
oops, it was a carry over from when the field was just `name`, it's been removed now